README.md

```diff
@@ -1,2 +1,31 @@
 # aws-synthetic-checks
+
 AWS re-usable synthetic uptime &amp; graphql operation checks stored in S3
+
+## Local Development
+
+### Fresh Setup
+
+Clone the repo:
```

Comment:
first thing tomorrow morning, i'll try running through these steps locally in a fresh clone to confirm.